governance/polls/Increase rETH Supply Cap to 60k - October 2, 2023.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Increase rETH Supply Cap to 60k - October 2, 2023
+summary: Increase rETH supply cap to 60k on SparkLend Ethereum.
+discussion_link: https://forum.makerdao.com/t/proposal-to-onboard-usdc-and-usdt-to-sparklend-ethereum/22227
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-10-02T16:00:00
+end_date: 2023-10-05T16:00:00
+
+
+---
+
+# Poll: Increase rETH Supply Cap to 60k - October 2, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, October 2 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+Express your support or opposition to increasing rETH supply cap from 20k to **60k** on SparkLend Ethereum.
```

Comment:
You could include this link to the Spark Protocol documentation: https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps

Reply:
Done.